### PR TITLE
Remove the remain MIN/MAX like macro

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary

Cotinue the work https://github.com/apache/nuttx/pull/8397

## Impact

Minor

## Testing

CI

### CODE REVIEW

**Review by gustavonihei — APPROVED** [review]

File: arch/sim/src/sim/posix/sim_alsa.c [hunk 1]
```diff
@@ -707,7 +701,7 @@ static void sim_audio_process(struct sim_audio_s *priv)
   else
     {
       expect = priv->playback ? apb->nbytes / priv->frame_size
-                              : AUDMIN(apb->nmaxbytes, priv->buffer_size)
+                              : MIN(apb->nmaxbytes, priv->buffer_size)
```
Comment (gustavonihei) [inline arch/sim/src/sim/posix/sim_alsa.c:704]: Need to include `sys/param.h` in this file.

File: drivers/lcd/ssd1351.c [hunk 2]
```diff
@@ -105,9 +105,7 @@
 
 /* Macro Helpers ************************************************************/
 
-#define SSD1351_MAX(a, b)        ((a) > (b) ? (a) : (b))
-#define SSD1351_MIN(a, b)        ((a) < (b) ? (a) : (b))
-#define SSD1351_CLAMP(n, a, b)   SSD1351_MIN(SSD1351_MAX(n, a), b)
+#define SSD1351_CLAMP(n, a, b)   MIN(MAX(n, a), b)
```
Comment (gustavonihei) [inline drivers/lcd/ssd1351.c:108]: Need to include `sys/param.h` in this file.

File: arch/sim/src/sim/posix/sim_alsa.c (hunk 1 above)
Comment (xiaoxiang781216) [reply arch/sim/src/sim/posix/sim_alsa.c:705]: Done.

File: drivers/lcd/ssd1351.c (hunk 2 above)
Comment (xiaoxiang781216) [reply drivers/lcd/ssd1351.c:109]: Done.

File: fs/partition/fs_gpt.c [hunk 3]
```diff
@@ -26,6 +26,7 @@
 #include <debug.h>
 #include <endian.h>
 #include <inttypes.h>
+#include <sys/params.h>
```
Comment (gustavonihei) [inline fs/partition/fs_gpt.c:29]:
```suggestion
#include <sys/param.h>
```

File: drivers/lcd/ssd1351.c [hunk 4]
```diff
@@ -24,6 +24,7 @@
 
 #include <nuttx/config.h>
 
+#include <sys/params.h>
```
Comment (gustavonihei) [inline drivers/lcd/ssd1351.c:27]:
```suggestion
#include <sys/param.h>
```

File: fs/partition/fs_gpt.c (hunk 3 above)
Comment (xiaoxiang781216) [reply fs/partition/fs_gpt.c]: Done.

File: drivers/lcd/ssd1351.c (hunk 4 above)
Comment (xiaoxiang781216) [reply drivers/lcd/ssd1351.c]: Done.

**Review by anchao — APPROVED** [review]

**Pull request merged**